electron-builder.json

```diff
@@ -1,6 +1,6 @@
 {
   "productName": "Slippi Launcher",
-  "appId": "gg.slippi.launcher",
+  "appId": "com.github.projectslippi.slippidesktopapp",
```

Comment:
`slippilauncher`, i'm going to change the repo name

Reply:
we can't change this app id anymore. it's what we're using to sign the macos stuff.

Reply:
ah rip